engine/crates/engine-v2/src/execution/coordinator.rs

```diff
@@ -43,12 +43,7 @@ impl<'ctx> ExecutorCoordinator<'ctx> {
     }
 
     pub async fn execute(&mut self) {
-        let mut futures = FuturesUnordered::<BoxFuture<'_, ExecutorFutResult>>::new();
-        // Mutation root fields need to be executed sequentially. So we're tracking for each
-        // executor whether it was for one and if so execute the next executor in the queue.
-        // Keeping the queue outside of the FuturesUnordered also ensures the future is static
-        // which wasm target somehow required. (not entirely sure why though)
-        let mut mutation_root_fields_executors = VecDeque::<Executor<'ctx>>::new();
+        let mut futures = FuturesUnordered::<BoxFuture<'_, ExecutorFutResult<'ctx>>>::new();
```

Comment:
removing a small hack I did earlier because I had a lifetime issue. Should have pushed that earlier.